browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1461,107 +1461,131 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		if (data.rawKeyEvents) {
 			var modifier = 0;
 
-			edit.addEventListener('keydown', function(event) {
-				if (edit.disabled)
-					return;
+			if (window.mode.isMobile() || window.mode.isTablet()) {
```

Comment:
here added different handlers for mobile, other changes are just indentation level change